CHANGELOG.md

```diff
@@ -1,6 +1,68 @@
-## 2.7.0 (on-going)
-
-> ????/??/??
+## 2.8.0 (on-going)
+
+### Tezos Integration:
+#### Keychain:
+- Tezos key derivation is based on Edward's curve (https://github.com/satoshilabs/slips/blob/master/slip-0010.md), 
+we don't have public derivations and since Libcore does not support private ones, only derivation scheme which is supported right now is `44'/<coin_type>'/<account_index>'/<node_index>'`,
+- Hence, resulting "derived" address from a keychain is directly a hash of the public key, 
+- Used hashing algorithm is `Blake2b`,
+- We support construction of accounts from both `Secp256k1` (`Base58`) extended public keys (prefixed with `xpub`) or from `Ed25519` (`Base58`) extended public keys (prefixed with `edpk`).
+#### Wallet logic: 
+- Originated accounts are considered as sub-accounts of implicit accounts that originated them,
+#### Explorer:
+- The explorer used by the default (for the moment) is Tzscan (used endpoint: https://api6.tzscan.io/v3),
+- **TODO**: we still miss some endpoints or could improve some others: 
```

Comment:
This has nothing to do in a changelog; this belongs to the issue tracker, or a roadmap. I think we should not pollute the changelog, that will be read by people who need to migrate their dependencies to a newer one, with things that are not done: it’s going to confuse them a lot and it will bring zero value to them.